.github/workflows/ci.yml

```diff
@@ -90,20 +90,40 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
 
-      - name: Determine tag
+      - name: Determine Docker Tag
         uses: haya14busa/action-cond@v1
         id: imagetag
         with:
           cond: ${{ github.event_name == 'pull_request' }}
           if_true: ${{ github.sha }}
-          if_false: "master"
+          if_false: 'master'
 
-      - name: Build
-        uses: docker/build-push-action@v1
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
         with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: dexidp/dex
-          tags: ${{ steps.imagetag.outputs.value }}
-          add_git_labels: true
-          push: ${{ github.event_name == 'push' }}
+          platforms: all
+
+      - name: Set up Docker Buildx
+        id: buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          install: true
+          version: latest
+          driver-opts: image=moby/buildkit:master
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      - name: Build and Push
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          file: Dockerfile
+          platforms: linux/amd64,linux/arm/v7,linux/arm64
+          push: true
```

Comment:
Please also add an equivalent to `add_git_labels`. See the action's README for details.

Reply:
We're using v2 action, I didn't see this flag as an available option: https://github.com/docker/build-push-action/blob/master/action.yml

Reply:
Just to confirm it's deprecated in v2 https://github.com/docker/build-push-action#complete-workflow

> If you come from v1 and you want an "automatic" tag management through Git reference and OCI Image Format Specification for labels, you will have to do it in a dedicated step for now.

Reply:
Yep, I meant adding those labels manually.

Reply:
@sagikazarmark which labels should be added? I didn't see any being included in the Dockerfile and I'm not quite sure where they're listed to be able to add them, if you could point me to the right direction this should be a small change.

Reply:
The ones listed in the readme is fine:

```
          labels: |
            org.opencontainers.image.title=${{ github.event.repository.name }}
            org.opencontainers.image.description=${{ github.event.repository.description }}
            org.opencontainers.image.url=${{ github.event.repository.html_url }}
            org.opencontainers.image.source=${{ github.event.repository.clone_url }}
            org.opencontainers.image.version=${{ steps.prep.outputs.version }}
            org.opencontainers.image.created=${{ steps.prep.outputs.created }}
            org.opencontainers.image.revision=${{ github.sha }}
            org.opencontainers.image.licenses=${{ github.event.repository.license.spdx_id }}
```

Reply:
Added